Cargo.toml

```
@@ -310,7 +310,7 @@ sources = [
   "sources-journald",
   "sources-kafka",
   "sources-kubernetes-logs",
-  "sources-logplex",
+  "sources-heroku-logs",
```

The common pattern here is to use underscores in the source name:
```suggestion
  "sources-heroku_logs",
```
This will of course have to change all the other instances below.